inference-engine/tests/functional/plugin/shared/include/behavior/config.hpp

```diff
@@ -174,9 +174,7 @@ namespace BehaviorTestsDefinitions {
         if (targetDevice.find(CommonTestUtils::DEVICE_GNA) != std::string::npos) {
             ASSERT_THROW(ie->SetConfig(configuration, targetDevice), InferenceEngine::NotFound);
         } else {
-            try {
```

Comment:
previoulsy the test incorrectly passed for MULTI (since the test wasn't checking for throwing)